src/Microsoft.TemplateEngine.Cli/Installer.cs

```diff
@@ -189,6 +200,23 @@ private void InstallRemotePackages(List<Package> packages)
             InstallLocalPackages(newLocalPackages);
         }
 
+        private void InstallGitSources(List<GitSource> gitSources)
+        {
+            _paths.CreateDirectory(_paths.User.ScratchDir);
+
+            List<string> newLocalPackages = new List<string>();
+
+            foreach (GitSource source in gitSources)
+            {
+                string targetPath = $"{_paths.User.ScratchDir}/{source.RepositoryName}";
+                ExecuteProcess("git", "clone", source.GitUrl, targetPath);
```

Comment:
Not a big fan of this, if you don't have git on the path, you can't use the feature. The alternative would be to embed a git client, but that's not very attractive either. I'd really rather get to a place where you can install installers, where this would light up for people who explicitly acquire it